src/SDKs/DataBox/Management.DataBox/Microsoft.Azure.Management.DataBox.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.DataBox</PackageId>
+		<Description>Microsoft Azure Management DataBox library</Description>
+		<AssemblyName>Microsoft.Azure.Management.DataBox</AssemblyName>
+		<Version>1.0.0-preview</Version>
+		<PackageTags>Microsoft Azure DataBox management;DataBox;DataBox management;</PackageTags>
+		<NoWarn>$(NoWarn);CS1591</NoWarn>
```

Comment:
@kushalvms is this warning coming from your project? if yes we are asking help of all RPs to take care of their own project and not to introduce this warning.